src/components/Sykmelding/Sykmeldingsperiode.tsx

```diff
@@ -13,7 +13,7 @@ import styles from './Sykmeldingsperiode.module.css';
 
 export interface PeriodeFormValue {
     sykmeldingstype: PeriodeType;
-    grad?: number | undefined;
+    grad?: string | undefined;
```

Comment:
Grad må vel vere number sidan inputfeltet er satt til type number, og så skal det vel vere int i backend?

Reply:
når jeg skrev testen så så jeg at typingen var brutt. Jeg tror `event.target.value` alltids er string selv om input-typen er number.

Reply:
mulig det gir mer mening å gi RHF ansvaret for å parse det til et tall da, man kan sette en `valueAsNumber`-prop på feltet:
![image](https://user-images.githubusercontent.com/1507032/196911735-270ea972-c48d-4998-8b1c-f80abc79750a.png)

Reply:
men det funker ikke med useController, kan teste om jeg kan bruke `register` siden dette er en "primitiv" input

Reply:
La til en ny commit du kan ta en titt på. Var litt mer kronglete å bruke register, men det lar i hvert fall RHF ta ansvar for å holde typingen og verdiene i sync.